src/bioregistry/gh/new_prefix.py

```diff
@@ -215,10 +215,13 @@ def make_title(prefixes: Sequence[str]) -> str:
 @click.command()
 @click.option("--dry", is_flag=True, help="Dry run - do not create any PRs")
 @click.option("--github", is_flag=True, help="Use this flag in a GHA setting to set run variables")
+@click.option("--issue", is_flag=False, help="Specific issue to process rather than finding all relevant ones")
```

Comment:
`is_flag=False` is the default - you can make this more explicit by setting `type=int`

```suggestion
@click.option("--issue", type=int, help="Specific issue to process rather than finding all relevant ones")
```